distributions/docker/src/main/docker/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM java:8-jdk
+COPY maven/${app.artifactId}-${project.version}.jar /opt/replication/app.jar
+COPY maven/demo-config/config /opt/replication/demo-config/config
+COPY maven/demo-config/keystores /opt/replication/demo-config/keystores
+COPY maven/demo-config/application.properties /opt/replication/demo-config/application.properties
+
+ENV truststore_pass changeit
+ENV truststore_type jks
+ENV truststore_file /opt/replication/demo-config/keystores/serverTruststore.jks
+ENV keystore_pass changeit
+ENV keystore_type jks
+ENV keystore_file /opt/replication/demo-config/keystores/serverKeystore.jks
+ENV config_dir /opt/replication/demo-config/
+ENV app_config_file /opt/replication/demo-config/application.properties
+
+ENTRYPOINT java \
+    -Djavax.net.ssl.trustStorePassword=${truststore_pass} \
+    -Djavax.net.ssl.keyStoreType=${truststore_type} \
+    -Djavax.net.ssl.trustStore=${truststore_file} \
+    -Djavax.net.ssl.keyStorePassword=${keystore_pass} \
+    -Djavax.net.ssl.trustStoreType=${keystore_type} \
+    -Djavax.net.ssl.keyStore=${keystore_file} \
+    -Duser.dir=${config_dir} \
```

Comment:
❓ I thought we we're setting these properties in the ssl.system.properties file? Could you maybe explain what's going on in this file? It seems intuitive but I don't want to make too many assumptions.